pcawg_metadata_parser/generate_release.py

```diff
@@ -110,13 +110,13 @@
                         #     ]
                         #   }
                         # },
-                        # {
-                        #   "terms": {
-                        #     "duplicated_bwa_alignment_summary.exists_mismatch_bwa_bams": [
-                        #       "T"
-                        #     ]
-                        #   }
-                        # },
+                        {
+                          "terms": {
+                            "duplicated_bwa_alignment_summary.exists_mismatch_bwa_bams": [
+                              "T"
+                            ]
+                          }
+                        },
```

Comment:
I wonder why this filter was turned off. Maybe it meant to be temporary. We must have this on when preparing real release.